test/method/testbader.py

```diff
@@ -56,3 +57,31 @@ def test_val(self):
         refData = [6.9378, 0.3639, 0.3827]  # values from `bader` package
 
         assert_allclose(self.analysis.fragcharges, refData, atol=0.15)
+
+    def test_chgsum_hf(self):
+        """Does the sum of charges equate to the number of electrons for a simple molecule?"""
+
+        hfpath = os.path.join(os.path.dirname(os.path.realpath(__file__)), "hf.out")
+        data = ccread(hfpath)
+        vol = volume.Volume((-6, -6, -6), (6, 6, 6), (0.2, 0.2, 0.2))
+        analysis = Bader(data, vol)
+        analysis.calculate()
+
+        self.assertAlmostEqual(numpy.sum(analysis.fragcharges), 10, delta=1)
+
+    def test_symms_benzene(self):
+        """ Do the carbons in benzene ring get assigned with roughly equal charges?"""
+
+        benzenepath = os.path.join(os.path.dirname(os.path.realpath(__file__)), "benzene.out")
+        data = ccread(benzenepath)
+        vol = volume.read_from_cube(
+            os.path.join(os.path.dirname(os.path.realpath(__file__)), "benzene.cube")
+        )
+        assert_allclose(vol.origin, numpy.array([-5, -5, -5]), atol=0.5)
```

Comment:
The value for each component of `vol.origin` is -5.05001036. Can you make both the reference value and the `atol` stricter?

Reply:
Sure -- I'll change this right away.